examples/LoRaWAN/LoRaWAN_End_Device/LoRaWAN_End_Device.ino

```diff
@@ -87,17 +87,28 @@ void setup() {
   // and can be set to NULL
 
   // some frequency bands only use a subset of the available channels
-  // you can set the starting channel and their number
-  // for example, the following corresponds to US915 FSB2 in TTN
+  // you can select the specific band or set the first channel and last channel
+  // for example, either of the following corresponds to US915 FSB2 in TTN
   /*
-    node.startChannel = 8;
-    node.numChannels = 8;
+    node.selectSubband(2);
+    node.selectSubband(8, 16);
   */
 
   // now we can start the activation
   // this can take up to 20 seconds, and requires a LoRaWAN gateway in range
+  // a specific starting-datarate can be selected in dynamic bands (e.g. EU868):
+  /* 
+    uint8_t joinDr = 8;
+    state = node.beginOTAA(joinEUI, devEUI, nwkKey, appKey, joinDr);
+  */
+  // a specific band can be selected for joining in fixed bands (e.g. US915):
+  /*
+    uint8_t subband = 2;
+    state = node.beginOTAA(joinEUI, devEUI, nwkKey, appKey, subband);
+  */
```

Comment:
I don't think it's a good idea to have a parameter whose interpretation changes based on the selected band. Also there already is a way to configure the subband, and that's via `selectSubband`.

I would propose to only use this parameter for `joinDr`, and ignore the value if it is set for fixed bands (where the user does not get to select the join DR).